meg_runtime/ui/manager.py

```diff
@@ -47,6 +60,58 @@ def get_instance(**kwargs):
             UIManager(**kwargs)
         return UIManager.__instance
 
+    @staticmethod
+    def open_about():
+        """Open the about menu."""
+        instance = UIManager.get_instance()
+        desc = ('<center>'
+                '<h1>Multimedia Extensible Git</h1>'
+                '<p>Version 0.1</p>'
+                '</center>')
+        QtWidgets.QMessageBox.about(instance, "About MEG", desc)
```

Comment:
This should use App.get_name() and App.get_version() (which needs created apparently). Also, the title should be `f'About {App.get_name()}'`.